src/test/regress/citus_tests/config.py

```diff
@@ -328,12 +328,6 @@ def __init__(self, arguments):
         self.new_settings = {"citus.shard_count": 1}
 
 
-class CitusNonMxClusterConfig(CitusMXBaseClusterConfig):
```

Comment:
Wouldn't it be useful to have one `CitusNonMxClusterConfig` test? In the end, we still support that.

Note that all other schedules are already extending `CitusMXBaseClusterConfig `.  Meaning that only `CitusNonMxClusterConfig` is non-mx, and it was intentional.

I'd have a slight preference to keep this schedule until we drop all the support for non-metadata synced clusters.

For now, adding some comment (similar to the the explanation I wrote above) would be good

Reply:
Or, maybe do the change in `CitusBaseClusterConfig`  but remove ` CitusMXBaseClusterConfig` class? That seems a lot easier to follow? And, still keep `CitusNonMxClusterConfig` which extends `CitusBaseClusterConfig`